docs/CSS.md

```diff
@@ -0,0 +1,125 @@
+### 1. `max-width`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `none` No max width, this is the default
+* `length` defines max width in px, cm, etc..
+* `%` defines max width in percent of containing block
+* `initial` sets this property to max value
+* `inherit` inherits this property from parent elemnt
+
+### 2. `position`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `static` Default value. Elements render in order
+* `absolute` element is positioned relativeto first positioned (not static) element
+* `fixed` element positioned relative to browser window
+* `relative` The element is positioned relative to its normal position
+* `inherit` inherits this property from its parent element
+
+### 3. `float`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `none` element is not floated and will be displayed where it is in text. default
+* `left` the element floats to the left
+* `initial` sets the property to its default value
+* `inherit` inherits this property from its parent
+
+### 4. `clear`
+
+* elements: all
+* inherited: yes
+
+#### Values
+
+* `none` default, allows floating elements on both sides
+* `left` no floating elements allowed on left side
+* `right` no floating elemenets allowed on right side
+* `both` no floating elements allowed on either the right or left side
+* `initial` sets this property to its default value
+* `inherit` inherits this property from its parent element
+
+### 5. `margin`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `length` Specifies a margin in px, pt, cm, etc.. default is 0
+* `%` spedifies a margin in percent width of the containing element
+* `auto` the browser calculates margin
+* `initial` sets this property to its default value
+* `inherit` inherits this property from its parent element
+
+### 6. `box-sizing`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `content-box` Default. The width and height properties and min and max includes only the content.  border, padding, margin not included
+* `border-box` width and height properties and min and max properties includes content padding and border but not margin
+* `initial` sets this property to its default value
+* `inherit` inherits this property from its parent element
+
+### 7. `padding`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `length` specifies the padding in px, pt, cm, etc. default is 0
+* `%` specifies the padding in percent of the width of the containing element
+* `initial` sets this property to its default value
+* `inherit` inherits this property from the parent element
+
+### 8. `border`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `border width` specifies the width of the border. default value is medium
+* `border style` specifies the style of the border. default is none
+* `border color` specifies the color of the border. default is color of element
+* `initial` sets this property to its default value
+* `inherit` inherits this property from its parent element
```

Comment:
Properties like `border`, `padding` and `margin` are _composite properties_ or shortcuts for several related properties. It's important to note how relative values -- like % -- work when applied to each of the individual properties. It can be pretty frustrating until you get a :bulb: moment.